### PR TITLE
Implement SPI 3.1 endpoint

### PR DESCRIPTION
**Pull request by ioannisNoukakis**
Implemented SPI 3.1 endpoint according to https://www.etsi.org/deliver/etsi_ts/102800_102899/102818/03.01.01_60/ts_102818v030101p.pdf

### CODE REVIEW

File: radiodns/apps/si_pi/views.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                                   ], service_provider.station_set.select_related().all()))
+
+    return render(
+        request,
+        "xml3.html",
+        context={
+            "service_provider": service_provider,
+            "stations": stations,
+            "service_provider_images": service_provider.logoimage_set.first(),
+            "media_root": "/media/"
```
Comment (pglimos) [inline radiodns/apps/si_pi/views.py:40]: How about we use the setting.MEDIA_ROOT instead of hardcoding?

Comment (pglimos) [reply radiodns/apps/si_pi/views.py:40]: And what is the response type that is needed xml, or html?

File: radiodns/config/urls.py [hunk 2]
```diff
@@ -34,11 +34,12 @@
     path("clients/", include("apps.clients.urls")),
     path("radiovis/", include("apps.radiovis.urls")),
     path("radioepg/", include("apps.radioepg.urls")),
+    path("radiodns/", include("apps.si_pi.urls")),
     path("system/", include("apps.system.urls")),
     path("admin/", admin.site.urls),
 ]
 #if settings.DEBUG:
-#    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (pglimos) [inline radiodns/config/urls.py:42]: the if debug is there for a reason. In the prod/staging we want nginx to be serving files.

**Comment by ioannisNoukakis** [comment]
Yup, anyways spi_pi files will be served from s3 instead like we did in the old implementation after talking with Ben so I'm going to close this and reopen with your feedback and s3.

**Pull request closed without merging**